include/AutomationEditor.h

```diff
@@ -240,6 +240,8 @@ protected slots:
 
 	EditModes m_editMode;
 
+	bool m_mouseDownLeft; //true if left click is being held down
```

Comment:
Since you end up only using `m_mouseDownRight` you should remove the `m_mouseDownLeft` as it's not used anywhere.